xl2times/utils.py

```diff
@@ -60,6 +60,12 @@ def explode(df, data_columns):
     :return:                Tuple with the exploded dataframe and a Series of the original
                             column name for each value in each new row.
     """
+    if df.columns.duplicated().any():
+        raise ValueError(
+            f"Dataframe has duplicated columns: {df.columns[df.columns.duplicated()]}"
+        )
+
```

Comment:
This is where I was hitting the duplicate column exception.  This check doesn't get used now the columns get merged earlier, but is probably still worth doing for other routes to `explode()`.